DZ Robot/src/by/teachmeskills/robot/Robot1.java

```diff
@@ -0,0 +1,56 @@
+package by.teachmeskills.robot;
+
+import by.teachmeskills.robot.hands.SamsungHand;
+import by.teachmeskills.robot.heads.SamsungHead;
+import by.teachmeskills.robot.legs.SonyLeg;
+
+public class Robot1 implements IRobot{
```

Comment:
You don't need to declare classes for combinations of robot parts, because they don't have any specific behavior. Just use the constructor of the Robot class
like
     Robot robot1 = new Robot(samsungHand, samsungHead, sonyLeg);
     Robot robot2 = new Robot(toshibaHand, toshibaHead, sonyLeg);
     Robot robot3 = new Robot(sonyHand, sonyHead, toshibaLeg);